sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/ir/expression/literal/DBSPBoolLiteral.java

```diff
@@ -71,8 +71,8 @@ public void accept(InnerVisitor visitor) {
     }
 
     @Override
-    public DBSPLiteral getNonNullable() {
-        return new DBSPBoolLiteral(Objects.requireNonNull(this.value));
+    public DBSPLiteral getWithNullable(boolean mayBeNull) {
```

Comment:
I'm not sure I follow this change. At least it's not clear from the names here.

Reply:
This is documented in the base class. 
In general I don't document functions that override - it would be too much work and a problem for maintaining them when something changes.